app/src/main/java/com/example/mydigitalcloset/clothingSeeAll.java

```diff
@@ -0,0 +1,103 @@
+package com.example.mydigitalcloset;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.mydigitalcloset.databinding.ActivityClothingSeeAllBinding;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.storage.StorageReference;
+
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+
+public class clothingSeeAll extends AppCompatActivity {
+
+
+    private DatabaseReference mDatabase;
+    ProgressDialog progressDialog;
+    ActivityClothingSeeAllBinding binding;
+    StorageReference storageReference;
+
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_clothing_see_all);
+
+        // Get a reference to the "clothing" node in your Firebase Realtime Database
+        mDatabase = FirebaseDatabase.getInstance().getReference().child("clothing");
+
+        Button topsButton = findViewById(R.id.seeTops);
+        topsButton.setOnClickListener(v -> {
+            // Find an existing clothing item from the"clothing/tops" node in your database
+            String clothingId = mDatabase.child("tops").push().getKey();
```

Comment:
make sure this & similar lines dont make new children. it might work better to create different variables for child references